traffic_portal/app/src/common/api/AuthService.js

```diff
@@ -57,6 +57,25 @@ var AuthService = function($rootScope, $http, $state, $location, $q, $state, htt
         return deferred.promise;
     };
 
+    this.oauthLogin = function(token) {
+        return httpService.post(ENV.api['root'] + 'user/login/oauth', { t: token})
+            .then(
+                function(result) {
+                    $rootScope.$broadcast('authService::oauthLogin');
+                    var redirect = decodeURIComponent($location.search().redirect);
```

Comment:
I'm finding myself saying this a lot: I know that we use this throughout the TP codebase, but for ease of debugging would you mind using `let`/`const` (`const` is appropriate in this case, I think) instead of `var`?

Reply:
done. and ill keep my eye out for more things like this as i continue to work in TP